### ISSUE TITLE
Boxplot enhancements

### ISSUE THREAD

**Issue by dannyparsons** [IDEMSInternational/R-Instat#3402]
1. Should we have x axis text angle on the main dialog (as well as themes)? Or could this be on a climatic boxplots dialog only?
2. Boxplots can be used when the x-variable is a variate - rather than a factor. Here is an example from the documentation `ggplot(diamonds, aes(carat, price)) + geom_boxplot(aes(group = cut_width(carat, 0.25)))`
So could we permit this (exceptionally in the main dialogue - or somewhere)? If a variate, then you need to group by the `cut_width,` so there would be an additional field and the default `cut_width` could be 1. I wonder if this could go on the boxplot options? Then the year could be a variate and I think the axis labels would then be fine.
3. When making the year into a factor it is logically an ordered factor. I feel there could usefully be an option with an ordered factor to not display all levels on the axis. I don't know whether this is indeed possible, or whether it makes a good question to ask?

From here #2084

**Comment by dannyparsons** [comment]
I do not believe 3. is possible in any "automatic" way other than setting the axis labels manually.

**Comment by rdstern** [comment]
The x-axis angle (or 90 degrees could usefully go on the x-axis sub-dialogue.  It would be done automatically on the climatic boxplots!
